LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -185,10 +179,27 @@ public bool HasPassword(User user)
             return true;
         }
 
-        /// <inheritdoc />
+        /// <summary>
+        /// Changes the users password (Requires privileged bind user).
+        /// </summary>
+        /// <param name="user">The user who's password will be changed.</param>
+        /// <param name="newPassword">The new password to set.</param>
+        /// <returns>Completed Task notification.</returns>
+        /// <exception cref="NotImplementedException">Thrown if AllowPassChange set to false.</exception>
         public Task ChangePassword(User user, string newPassword)
         {
-            throw new NotImplementedException();
+            if (!LdapPlugin.Instance.Configuration.AllowPassChange)
+            {
+                throw new NotImplementedException();
+            }
+
+            var passAttr = LdapPlugin.Instance.Configuration.LdapPasswordAttribute ?? "userPassword";
```

Comment:
I would prefer to throw an exception here if the attribute isn't set


```c#
if (string.IsNullOrEmpty(LdapPlugin.Instance.Configuration.LdapPasswordAttribute))
{
   throw new InvalidOperationException("Password attribute is not set");
}
```

Reply:
Done - [1cb23a7](https://github.com/jellyfin/jellyfin-plugin-ldapauth/pull/115/commits/1cb23a7403c434fcb50443e1533a8d451a9788c5)

Reply:
FYI, with the null check being added, I removed the default attribute, "userPassword", at the end of LN196, since it will never be used.